numba/targets/arraymath.py

```diff
@@ -1413,6 +1413,33 @@ def np_tril_impl_multi(m, k=0):
     else:
         return np_tril_impl_multi
 
+@overload(np.tril_indices)
+def np_tril_indices(n, k=0, m=None):
+
+    # we require integer arguments, unlike numpy
+    if not isinstance(n, (int, types.Integer)):
+        raise TypeError('n must be an integer')
+    if not isinstance(k, (int, types.Integer)):
+        raise TypeError('k must be an integer')
+    if not (m is None or isinstance(m, (int, types.Integer))):
+        raise TypeError('m must be an integer')
+
+    def np_tril_indices_impl(n, k=0, m=None):
+        return np.nonzero(np.tri(n, m, k=k))
```

Comment:
Just a question here: the NumPy docs state that 'By default m is taken equal to n.'. Will this happen in the overload implementation of `np.tri`?

Reply:
Yes "By default, M is taken equal to N"
https://docs.scipy.org/doc/numpy/reference/generated/numpy.tri.html

Reply:
Indeed, this is what the docs say. Do you happen to know where Numba does this?

Reply:
Yes https://github.com/numba/numba/blob/master/numba/targets/arraymath.py#L1360-L1361